tslint.json

```diff
@@ -44,6 +44,7 @@
         "type-operator-spacing": true,
         "prefer-const": true,
         "no-in-operator": true,
-        "no-increment-decrement": true
+        "no-increment-decrement": true,
+        "object-literal-surrounding-space": true

```

Comment:
That carries the weight of supporting all of the eslint rule's options, which this does not.